extensions/blocks/rating-star/rating-meta.php

```diff
@@ -74,7 +74,13 @@ function jetpack_rating_meta_get_symbols( $attributes ) {
 		// These are hidden by default, then unhid when CSS loads.
 		$symbols_hifi = array();
 		for ( $pos = 1; $pos <= $attributes['maxRating']; $pos++ ) {
-			$symbols_hifi[] = '<span style="display: none;">' . jetpack_rating_meta_get_symbol_high_fidelity( $attributes, $pos ) . '</span>';
+			$schema = '';
```

Comment:
For better readability, can we extract this new code to a function along the lines of the following?

> $schema = get_schema_for_item( $pos, $attributes['maxRating'] ):

Reply:
Thanks for the suggestion, just pushed this update